modules/nf-core/hisat2/build/main.nf

```diff
@@ -10,13 +10,13 @@ process HISAT2_BUILD {
         'biocontainers/hisat2:2.2.1--h1b792b2_3' }"
 
     input:
-    path fasta
+    tuple val(meta), path(fasta)
     path gtf
```

Comment:
Don't we also need `meta` here?

Reply:
Ha.. good catch! Fixing it right away 👍🏻